Move updates from 1.16 to 1.18 branch

remplerus opened the pull request.

remplerus committed:
Update gradle.yml

remplerus committed:
Update gradle.yml

remplerus committed:
Delete .github/workflows directory

remplerus committed:
Merge branch 'NovaMachina-Mods:1.16' into 1.16

ModernGamingWorld committed:
Updated Forge to Match June 16th Forge Version

ModernGamingWorld committed:
Merge branch 'NovaMachina-Mods:1.16' into 1.16

ModernGamingWorld committed:
Updated Craft Tweaker Version

Updated Craft Tweaker Version

ModernGamingWorld committed:
Merge branch 'NovaMachina-Mods:1.16' into 1.16

ModernGamingWorld committed:
Update gradle.properties

ModernGamingWorld committed:
Merge branch '1.16' of https://github.com/ModernGamingWorld/ExNihiloSequentia into 1.16

ModernGamingWorld committed:
Merge branch 'NovaMachina-Mods:1.16' into 1.16

ModernGamingWorld committed:
Updated Craft Tweaker

NovaMachina committed:
Fix #240 Fix AB#65 Regenerated recipes

NovaMachina committed:
Prep 2.0.2.9

NovaMachina committed:
Update PR workflow

NovaMachina committed:
Merge branch '1.16-dev' into 1.16

NovaMachina committed:
Remove debug statement for curseforge task

NovaMachina committed:
Merge pull request #227 from ModernGamingWorld/1.16

Updated Forge to Match June 16th Forge Version

ModernGamingWorld committed:
Merge branch 'NovaMachina-Mods:1.16' into 1.16

NovaMachina committed:
Merge pull request #236 from ModernGamingWorld/1.16

Updated Craft Tweaker Version

ModernGamingWorld committed:
Updated Craft tweaker

ModernGamingWorld committed:
Merge branch '1.16' of https://github.com/ModernGamingWorld/ExNihiloSequentia into 1.16

ModernGamingWorld committed:
Updated CraftTweaker to 7.1.0.326

ModernGamingWorld committed:
Updated Craft Tweaker to 7.1.0.327

ModernGamingWorld committed:
Updated Craft Tweaker to 7.1.0.328

ModernGamingWorld committed:
Updated Crafttweaker to 7.1.0.329

ModernGamingWorld committed:
Updated Craft tweaker to 7.1.0.330

ModernGamingWorld committed:
Updated Craft Tweaker to 7.1.0.331

ModernGamingWorld committed:
Updated JEI to 7.7.0.107

ModernGamingWorld committed:
Updated Craft Tweaker to 7.1.0.334

ModernGamingWorld committed:
Updated Craf tweaker to 7.1.0.337

ModernGamingWorld committed:
Updated Craft Tweaker to 7.1.0.342

ModernGamingWorld committed:
Update gradle.properties

ModernGamingWorld committed:
Updated JEI to 7.7.1.110

ModernGamingWorld committed:
Updated Forge to 36.1.33

ModernGamingWorld committed:
Updated Craftwaker to 7.1.0.350

ModernGamingWorld committed:
Updated Forge to 36.1.35

remplerus committed:
render any block in sieve

ModernGamingWorld committed:
Updated Forge Build To 1.16.5-36.1.36

ModernGamingWorld committed:
Updated Forge Build To 1.16.5-36.1.53

Check to make sure it does not crash, it should not tho

ModernGamingWorld committed:
Updated Forge Build To 1.16.5-36.1.58

ModernGamingWorld committed:
Updated Forge Build To 1.16.5-36.1.61

ModernGamingWorld committed:
Updated license under Mods.toml to follow your license you have

Please check to make sure I did this right

ModernGamingWorld committed:
Added Jade Support

ModernGamingWorld committed:
Updated 1.16.5-36.1.65

ModernGamingWorld committed:
Updated to Forge New 1.16.5 RB 36.2.0

remplerus committed:
ctm-fix?

remplerus committed:
updated getting silkworm and string as datagen

ModernGamingWorld committed:
Updated JEI Version to 7.7.1.111

NovaMachina committed:
Merge pull request #247 from remplerus/maybe-fix-ctm-issue

Fix CTM issue

NovaMachina committed:
Merge pull request #245 from remplerus/sieve-rendering

render any block in sieve

NovaMachina committed:
Merge branch '1.16-dev' into 1.16

remplerus committed:
Squashed commit of the following:

commit 0990729fc0e416bab214f22d0c7fa4870a336ef2
Merge: e2475c6 5cfaba4
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon Jun 28 18:31:36 2021 -0500

    Merge pull request #236 from ModernGamingWorld/1.16

    Updated Craft Tweaker Version

commit 5cfaba4a417a1bec011e60065726ba09cab144e7
Merge: af8a669 e2475c6
Author: ModernGamingWorld <75923060+ModernGamingWorld@users.noreply.github.com>
Date:   Mon Jun 28 18:30:44 2021 -0500

    Merge branch 'NovaMachina-Mods:1.16' into 1.16

commit e2475c64b7e55c0152cd0120a8a41b220da81f6e
Merge: 939a22c c6f57c0
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon Jun 28 18:21:04 2021 -0500

    Merge branch '1.16-dev' into 1.16

commit c6f57c0125b27bad64fa276e9fa2c46982865e67
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon Jun 28 18:19:17 2021 -0500

    Update PR workflow

commit e637fb975273f949d60f1d9b5b988240ae4101fd
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon Jun 28 18:17:55 2021 -0500

    Prep 2.0.2.9

commit ec88b6e57d3d9b1001e0d34739c1768431ef7226
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon Jun 28 18:14:36 2021 -0500

    Fix #240 Fix AB#65 Regenerated recipes

commit af8a6695d29776048425743bd8e8f02650265c46
Author: ModernGamingWorld <ModernGamingWorld@outlook.com>
Date:   Mon Jun 28 17:14:44 2021 -0500

    Updated Craft Tweaker

commit e749ad81c292015ede19e342cf66a90166b8ab45
Merge: aa0a8e5 939a22c
Author: ModernGamingWorld <75923060+ModernGamingWorld@users.noreply.github.com>
Date:   Sat Jun 26 19:00:10 2021 -0500

    Merge branch 'NovaMachina-Mods:1.16' into 1.16

commit aa0a8e56ef6b052dd33c3275c0d039834e267dfb
Merge: 911c408 659ab8f
Author: ModernGamingWorld <ModernGamingWorld@outlook.com>
Date:   Sat Jun 26 18:59:46 2021 -0500

    Merge branch '1.16' of https://github.com/ModernGamingWorld/ExNihiloSequentia into 1.16

commit 911c408b6d81ba5a57db25c74c02d236f2555e82
Author: ModernGamingWorld <ModernGamingWorld@outlook.com>
Date:   Sat Jun 26 18:59:38 2021 -0500

    Update gradle.properties

commit 939a22cf17a4c8a35cdc14e0ac13a9e8609cf4de
Merge: 66aa6f5 1328e69
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Sat Jun 26 11:37:20 2021 -0500

    Merge pull request #239 from NovaMachina-Mods/1.16-dev

    1.16 dev

commit 1328e6969d00cf159bdd9ca70402779e8330dbda
Merge: f0e0b7d b2c748a
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Sat Jun 26 11:32:24 2021 -0500

    Merge pull request #238 from NovaMachina-Mods/azure-pipelines

    Azure pipelines

commit b2c748aca86ceab19f0d35cd7d4b5a63bab2d5d1
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Sat Jun 26 11:30:45 2021 -0500

    Update pipeline

commit d61432828747b57dfccb95f2d4af86ded076e6f1
Merge: a686001 d6d6839
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Sat Jun 26 11:29:34 2021 -0500

    Merge branch 'azure-pipelines' of github.com:NovaMachina-Mods/ExNihiloSequentia into azure-pipelines

commit f0e0b7d11ed534ced5bcb1809836d62422e70ab8
Merge: 9f81ff9 66aa6f5
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Sat Jun 26 11:25:30 2021 -0500

    Merge branch '1.16' into 1.16-dev

commit 9f81ff9ba63f5c9068ddee404a6155ad62f87843
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Sat Jun 26 11:22:50 2021 -0500

    Switch to timestamp versioning

    [Ticket: AB#59]

commit 66aa6f52324105ea4bbfdee443b75b8b182b972d
Merge: 2773dc5 d6d6839
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Sat Jun 26 09:52:01 2021 -0500

    Merge pull request #237 from NovaMachina-Mods/azure-pipelines

    Set up CI with Azure Pipelines

commit d6d683919de11238073f52820664170af0d4c9c3
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Sat Jun 26 09:09:01 2021 -0500

    Set up CI with Azure Pipelines

    [skip ci]

commit 2773dc5f84637453068ca8cdcfea2a2a9aaa3d40
Merge: 60ae96b d38258a
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Sat Jun 26 08:50:49 2021 -0500

    Merge branch '1.16' of github.com:NovaMachina-Mods/ExNihiloSequentia into 1.16

commit 60ae96ba60fc5a922881ad8216f38f2aefa86a0f
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Sat Jun 26 08:50:14 2021 -0500

    Update README

commit 659ab8f76fc5a7febee363027152ebdc2d12fd2b
Merge: 6e6253e d38258a
Author: ModernGamingWorld <75923060+ModernGamingWorld@users.noreply.github.com>
Date:   Fri Jun 25 18:22:12 2021 -0500

    Merge branch 'NovaMachina-Mods:1.16' into 1.16

commit 6e6253ed520bf83f5142eceb90f5f21df4161434
Author: ModernGamingWorld <ModernGamingWorld@outlook.com>
Date:   Fri Jun 25 18:21:33 2021 -0500

    Updated Craft Tweaker Version

    Updated Craft Tweaker Version

commit d38258ac63ee80166d6802a0ca6616d13acc82c3
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Fri Jun 25 12:29:59 2021 -0500

    Update issue-to-work-item.yml

commit c50216c299b50e871ee15f8543c230eaece0bf3c
Merge: 15b27ca 3e4ca07
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Thu Jun 24 17:05:00 2021 -0500

    Merge pull request #231 from NovaMachina-Mods/azure-boards

    Add the Azure Boards badge to the README

commit 15b27ca3fff7b8663dea7f6df1e6fd7ca30ba611
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Thu Jun 24 17:03:22 2021 -0500

    Create pr-to-work-item.yml

commit 8abfb5c6f9f5db6a3505f4fd2be417cef064a366
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Thu Jun 24 16:58:40 2021 -0500

    Update issue-to-work-item.yml

commit 9385107fa8ab42cb4fa7e85179f17af832f6394a
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Thu Jun 24 16:54:49 2021 -0500

    Update issue-to-work-item.yml

commit f6dd9edbcba07f99a594967fed458eaf38c51368
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Thu Jun 24 16:52:59 2021 -0500

    Update issue-to-work-item.yml

commit 77547068f08fdbe552d3b27b3cc621c47c65316f
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Thu Jun 24 16:49:24 2021 -0500

    Update issue-to-work-item.yml

commit 901da42bb9d5a7bb0ade6ad295c8273d01ddc264
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Thu Jun 24 16:47:40 2021 -0500

    Move workflow folder

commit a9f1d8ba3938f96c8cd2f96fc79899e15fa5be45
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Thu Jun 24 16:37:56 2021 -0500

    Add workflow for issues to work items

commit eaf7ccea53bbb4c46b53bf39003235b01bbab5ac
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Thu Jun 24 16:35:19 2021 -0500

    Delete issue-to-work-item.yml

commit 9d7bc72b89fee5bf45ce5167635f2d6cc75cf345
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Thu Jun 24 16:32:47 2021 -0500

    Create issue-to-work-item.yml

commit 3e4ca070caffa157d6c3aec342e15ee187b56860
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Wed Jun 23 19:48:10 2021 -0500

    Add the Azure Boards badge to the README. Fixes AB#5

commit a68600199654dfc182ce2e077fb9372539583309
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Thu Jun 17 10:20:02 2021 -0500

    Set up CI with Azure Pipelines

    [skip ci]

commit 754a49eddc1f411655fb1cdfb7c7d75f270351b1
Merge: 25b1763 6f43c99
Author: ModernGamingWorld <75923060+ModernGamingWorld@users.noreply.github.com>
Date:   Wed Jun 16 21:36:50 2021 -0500

    Merge branch 'NovaMachina-Mods:1.16' into 1.16

commit 25b17636074f0c6d87910eac49f1b1a47b297b59
Author: ModernGamingWorld <ModernGamingWorld@outlook.com>
Date:   Wed Jun 16 21:36:19 2021 -0500

    Updated Forge to Match June 16th Forge Version

commit 6f43c99fc7059cf680e57cb3996e9549d58ee035
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Wed Jun 16 10:58:58 2021 -0500

    Prep 2.0.2.8

commit 436bdffbcefaf75b08ee827a86f2a1f11aebff03
Merge: 2c9d7b2 693f7b4
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Sat Jun 12 20:53:16 2021 -0500

    Merge pull request #225 from lazynessmind/mesh-stackable

    Implemented #80: Mesh Stack Config

commit 2c9d7b2c9b610f07e64093bd6e5f8f4643de33e5
Merge: 845a4b6 bcea31e
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Sat Jun 12 20:50:49 2021 -0500

    Merge pull request #223 from yulinlin404/patch-1

    Update zh_cn.json

commit 845a4b635e4dc0977ab060f0bcdf5f4ecf1ff62d
Merge: 50e2dd1 78cbfab
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Sat Jun 12 20:49:24 2021 -0500

    Merge pull request #222 from remplerus/fix-#216

    Fix #216

commit 50e2dd14a8e1f7ef5c711636debb43cc35b591e1
Merge: 75f8ae1 6ca4e27
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Sat Jun 12 20:48:35 2021 -0500

    Merge pull request #221 from bigenergy/patch-1

    Added russian translation

commit 75f8ae1471214fdcab88a6fe9e5e4a202710cb2b
Merge: f286851 a25a063
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Sat Jun 12 20:46:23 2021 -0500

    Merge pull request #220 from ModernGamingWorld/1.16

    Updated Forge to match June 8th 1.16.5 Updates

commit 693f7b49b3190bf8bdc49fd3166b103cd15f9f74
Author: lazynessmind <lazynessmind@gmail.com>
Date:   Sat Jun 12 19:46:51 2021 +0100

    Make player able to stack the meshes.

commit f286851de25a64dd70a4577eee0020c48b398bb5
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Fri Jun 11 21:53:24 2021 +0200

    fixed #224

commit bcea31e3253eb202bf1ae4fcda3f43a3f1223ad8
Author: yulinlin404 <49760859+yulinlin404@users.noreply.github.com>
Date:   Fri Jun 11 15:12:57 2021 +0800

    Update zh_cn.json

commit 78cbfab5cd62d95ec1e16e4a82407b202c6f63d4
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Wed Jun 9 20:00:47 2021 +0200

    Update Config.java

commit f0dcc547face474880eb15f4b36996c8ed5b550d
Merge: a57efa2 3e5d600
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Wed Jun 9 19:58:16 2021 +0200

    Merge branch 'NovaMachina-Mods:1.16-dev' into 1.16-dev

commit 6ca4e274ee0d2fa0feba5a646b194f6ec3088a2b
Author: Big Energy <andre90094@gmail.com>
Date:   Wed Jun 9 06:19:28 2021 +0300

    Added russian translation

commit a25a063a5572e1841c0e9a43999ae9a9bfcf13c6
Author: ModernGamingWorld <ModernGamingWorld@outlook.com>
Date:   Tue Jun 8 17:52:39 2021 -0500

    Updated Forge to match June 8th 1.16.5 Updates

    Updated Forge to match June 8th 1.16.5 Updates

commit 7841be72465964f3d0b31a12cc8e747c48516b32
Merge: 3e5d600 177b0b8
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon Jun 7 08:45:55 2021 -0500

    Merge branch '1.16' of github.com:NovaMachina-Mods/ExNihiloSequentia into 1.16

commit 3e5d60090df0932d8a6ec178c5e0a5340cf1e816
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon Jun 7 08:43:12 2021 -0500

    Prep 2.0.2.7

commit a131652a1eb1876dd927e452e2933ad3c086abc8
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon Jun 7 08:35:06 2021 -0500

    Add Modern's PR that I merged to wrong branch

commit 177b0b8227a7ca3542e89441d75addd80c3e5ef0
Merge: 2396ef3 89d04a3
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon Jun 7 08:28:29 2021 -0500

    Merge pull request #218 from NovaMachina-Mods/revert-215-1.16

    Revert "Updated TOP & Craft tweaker"

commit 89d04a3b75dd1df34b7ff890219e59e946243fe0
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon Jun 7 08:26:24 2021 -0500

    Revert "Updated TOP & Craft tweaker"

commit 2396ef38026e9c63ad13b0053af25e73c7a9b6fd
Merge: 9b29e67 e2890e4
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon Jun 7 08:26:21 2021 -0500

    Merge pull request #215 from ModernGamingWorld/1.16

    Updated TOP & Craft tweaker

commit 0884c3826d3b5320ae26699b70cdbc8896997d8d
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon Jun 7 08:23:47 2021 -0500

    Update license to CC BY-NC-SA 4.0

    This is to protect the artists and those that have worked on Ex Nihilo:
    Sequentia so that they at least get attributed for their work.

commit bc9439ed6234187b3ba283b0103e9d4e7213bf82
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon Jun 7 08:13:00 2021 -0500

    Fix #132 Add TOP level

    TOP must be set to extended in order for certain values to be seen

    [Ticket: 132]

commit e2890e402c94269983fd3d3daff9bac750ead0db
Author: ModernGamingWorld <ModernGamingWorld@outlook.com>
Date:   Fri Jun 4 10:41:20 2021 -0500

    Delete fr_fr.json

commit f8cb46cdf9c4763927aa897a72ef62f5e1ee043f
Author: ModernGamingWorld <ModernGamingWorld@outlook.com>
Date:   Thu Jun 3 22:59:46 2021 -0500

    Added Japanese Translations

commit 6295df3c05a3d555ce6a9a6b88b16e17febaff30
Author: ModernGamingWorld <ModernGamingWorld@outlook.com>
Date:   Thu Jun 3 22:43:27 2021 -0500

    Added French Translation

commit 14440c245ebd9f00d05480a93ec55e581c27c103
Author: ModernGamingWorld <ModernGamingWorld@outlook.com>
Date:   Thu Jun 3 18:41:33 2021 -0500

    Updated Forge to 36.1.25, Matches June 3rd Forge updated for 1.16.5

commit f359087bf2d279ca2d4c6b8fafd472f64b326291
Author: ModernGamingWorld <ModernGamingWorld@outlook.com>
Date:   Wed Jun 2 19:57:26 2021 -0500

    Updated JEI Version to Match Update on June 2nd

commit f02459566cb40186846e487dcda71a058cb4291c
Merge: 4ca82a4 9b29e67
Author: ModernGamingWorld <75923060+ModernGamingWorld@users.noreply.github.com>
Date:   Tue Jun 1 22:53:14 2021 -0500

    Merge branch 'NovaMachina-Mods:1.16' into 1.16

commit 4ca82a453909fdeeb65b1ad6ebfdec7a90773a34
Author: ModernGamingWorld <ModernGamingWorld@outlook.com>
Date:   Tue Jun 1 22:52:53 2021 -0500

    Updated Both TOP Versions and Craft Tweaker Versions

commit 3f54b4976d75c9ce40f874e256573819b2a78cc9
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon May 31 09:21:07 2021 -0500

    Fix #154 Add config to set number of sieve clicks

    Allows for the setting of the number of clicks that it takes to sieve a
    block.

    [Ticket: 154]

commit b1e4bf037da7f347556a13dc3b291dd8f841d5a7
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon May 31 09:11:46 2021 -0500

    Fix #165 Allow infested leaves to be sheared.

    i[Ticket: 165]

commit b89264687b625e73eae08ba45206f468a9ee4540
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon May 31 08:53:58 2021 -0500

    Fix #199 Scale heat amount based on fluid level

    Heat amounts will be proportional to the fluid levels of flowing fluids.

    [Ticket: 199]

commit fae15eedeecd320062169b292ec0f4db0d606fd6
Merge: 1d34990 9b29e67
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon May 31 08:09:32 2021 -0500

    Merge branch '1.16' into 1.16-dev

commit 1d34990d542586d9c7464e2f46c165b2a89a3ae1
Merge: 71de027 d2824b9
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon May 31 08:06:04 2021 -0500

    Merge branch '1.16' into 1.16-dev

commit 9b29e6721203b790860fa98416abb6e284e02ccc
Merge: 633f5ea a01f0e4
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon May 31 07:59:32 2021 -0500

    Merge pull request #211 from ModernGamingWorld/1.16

    Updated the Following

commit 71de027dff8ef8ba772cb7f02bfa1d77ec9b47b5
Merge: 4598f9e 836897f
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon May 31 07:47:52 2021 -0500

    Merge pull request #207 from remplerus/add-wood-as-fuel

    add woods as fuel

commit 836897ffd8cecf173258671e3a35a77a151a7780
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Sun May 30 09:45:58 2021 +0200

    Update EnumResource.java

commit a01f0e4ef33791f6a3f2a9adabdbd11e47662ede
Author: ModernGamingWorld <ModernGamingWorld@outlook.com>
Date:   Wed May 26 22:28:10 2021 -0500

    Update gradle.properties

    Updated Forge as Ex Nihilo was Behind a few Updates

commit e6e62871a20e8991a9f06009ebe38c7db32bd8ca
Author: ModernGamingWorld <ModernGamingWorld@outlook.com>
Date:   Wed May 26 17:47:45 2021 -0500

    Update gradle.properties

    Updated JEI Version

commit 4598f9e55ee40fd452afbb761e4794a4a3443d53
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Wed May 26 00:03:20 2021 -0500

    Fix #203 Allow for ENS fluids to be dispensed

    [Ticket: 203]

commit 7efd519045318dd5f2f2c86e62ac58c9b430781c
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Tue May 25 22:20:57 2021 -0500

    Fix #186 Rearange code so that it works as expected

    [Ticket: 186]

commit 9187e95c14f9fa7cf7a48680688ca646435fd396
Merge: a80812b 4f61ad4
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Tue May 25 22:07:33 2021 -0500

    Merge pull request #209 from remplerus/revert-sieve-recipe

    Revert sieve recipe

commit 633f5eadfd60c7b7c5338a7b1bc6d65f0a364ba7
Merge: 2349770 b0d08a3
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Tue May 25 22:02:31 2021 -0500

    Merge pull request #204 from ModernGamingWorld/1.16

    Update gradle.properties

commit 2349770c74e7808cffedc67656b395c42161ad8c
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Tue May 25 21:53:48 2021 -0500

    Update CODEOWNERS

commit a57efa2cfda57385bd8e458572921c60c0545511
Merge: ae8120c a80812b
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Tue May 25 10:59:19 2021 +0200

    Merge branch 'NovaMachina-Mods:1.16-dev' into 1.16-dev

commit b0d08a3b9110e3462a16631333cdc937b5288d7e
Author: ModernGamingWorld <ModernGamingWorld@outlook.com>
Date:   Mon May 24 22:11:33 2021 -0500

    Update gradle.properties

    Updated JEI & Craft Tweaker version, Note you may need to make sure they dont cause issues

commit 4f61ad460df6a7ff2e9c510ee291fcf9164d2b8c
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Sat May 22 10:47:12 2021 +0200

    revert sieve recipe

commit 2285b3472419a61b7d6b97d93364e5852d47056b
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Sat May 22 10:43:51 2021 +0200

    updated mesh burning and burntime

commit 11c5a2adb39d1f7159f19789881444798ec67283
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Thu May 20 23:14:57 2021 +0200

    add woods as fuel

commit a80812bb8da0086f2b295961fee7b4d08f5982e5
Merge: bac760f d7dd338
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Thu May 20 09:07:43 2021 -0500

    Merge branch '1.16-dev' of github.com:NovaMachina-Mods/ExNihiloSequentia into 1.16-dev

commit bac760f70a5520f2f4b29bde8849c75c24cafeab
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Thu May 20 09:05:32 2021 -0500

    Remove some dangling code from Recipe Generator

commit d7dd3384069a4be73c2c6cd2527a5d4fc27cf72c
Merge: 12950de 341f4a3
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Thu May 20 09:04:29 2021 -0500

    Merge pull request #206 from remplerus/new-wood-blocks

    [FINAL VERSION] add new wooden blocks

commit 341f4a3a68cb5af57b43966ff3895d22dee9018c
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Thu May 20 13:28:11 2021 +0200

    revert oak to fit in 1.16

    names are easy to change for breaking 1.17 block name changes

commit b5f253f75a42a5086d336c4f9e2ee432f726cd98
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Thu May 20 13:15:43 2021 +0200

    update 2

commit 049d8adc7eec8edf694ae0a9d2f80fcd51fd931f
Author: ModernGamingWorld <ModernGamingWorld@outlook.com>
Date:   Wed May 19 22:57:49 2021 -0500

    Update gradle.properties

    Updated JEI & Craft Tweaker Versions

commit d9124e2dda95babad2d72ff35c35e479ce9c0d45
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Wed May 19 18:01:01 2021 +0200

    imp1

commit 12950de852aa722d2ebeebd2e619a8ffb0dbe751
Merge: 2562d92 b513cba
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon May 17 22:19:42 2021 -0500

    Merge pull request #198 from remplerus/fix-#196

    Add #196

commit b513cba6a539ffdf68c2a34fc3ab28c36156768f
Merge: 8ce514c 2562d92
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon May 17 22:19:29 2021 -0500

    Merge branch '1.16-dev' into fix-#196

commit 2562d9251442bbcdb0b9c2cb749da18bd094c6dc
Merge: 4df03ca 9efe339
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon May 17 22:08:12 2021 -0500

    Merge pull request #197 from remplerus/make-easy-addons

    Moved Recipe and Blockstate Creations from normal to abstract for later addon creation

commit 4df03cadccc40d9b6672b944810d7c1b06356137
Merge: 4074f30 fba59ef
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Mon May 17 22:05:55 2021 -0500

    Merge pull request #194 from remplerus/carryon-fix-maybe

    fix dupe with carryon

commit 8ce514c5ad590b571560d33c68464bfe66dd266f
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Mon May 10 19:23:37 2021 +0200

    add #196

commit 9efe3395cc94faa84195975ebc0dcbad40bbc025
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Mon May 10 19:17:18 2021 +0200

    make-easy-addons

commit ae8120cc9eedc7278107d1b4a025bf82595dc502
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Mon May 10 18:40:10 2021 +0200

    Delete .github/workflows directory

commit 5b5c591261dfa2da31ba3c103477fc791dcbe07c
Merge: 668d41d 4074f30
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Mon May 10 18:39:42 2021 +0200

    Merge branch 'NovaMachina-Mods:1.16-dev' into 1.16-dev

commit fba59ef8aca37cd332a455c1107cf4ff3d9b0ad0
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Fri May 7 16:01:52 2021 +0200

    fix some carry on issues

commit 4074f30089a964e8efedf359d57e574fac8ce83a
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Sun May 2 16:03:33 2021 -0500

    Prevent Mob spawn on infested leaves.

    It was possilbe for mobs to spawn on infested leaves. This has been
    fixed by adding a conditional to the infested leaves.

    [Ticket: ENS-29]

commit 8058cf280fe1072bed98e342c7eb5d983d72a32e
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Sun May 2 15:49:44 2021 -0500

    Balance Ancient Debris Recipe

    Ancient Debris recipe was way too expensive. Redueced it to a
    shapeless recipe that takes 1 Obsidian and 1 Netherite Scrap.

    [Ticket: ENS-5]

commit 668d41d31010510c3e50dd2d0db4684caaec4fe2
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Tue Mar 30 13:18:22 2021 +0200

    Update gradle.yml

commit 1a1d21d75febde4215e33bd0a572b17dc922fd85
Merge: 04d4913 1ae4b48
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Tue Mar 30 13:16:36 2021 +0200

    Merge pull request #3 from NovaMachina-Mods/1.16-dev

    standard 1.16-dev

remplerus committed:
Merge branch 'NovaMachina-Mods:1.16' into 1.16

remplerus committed:
updated bunch of things

- Update to FG5, Gradle 7
- Update Forge to 36.2.2
- Update CT to 7.1.0.364
- Update TOP to 3.1.4-22
- Update JEI to 7.7.1.116
- Add new block: Stone Crafting Table
- Make Stone great again: break clean stone with bare hand or with crook to get pebbles (make stoneblock-related modpacks easier to create)
- Update BurnTime function to use not-deprecated function

NovaMachina committed:
Merge pull request #241 from ModernGamingWorld/1.16

Updated Craft Tweaker

NovaMachina committed:
Merge branch '1.16-dev' into update-to-FG5

NovaMachina committed:
Merge pull request #252 from remplerus/update-to-FG5

Update to FG5, Update Mods, Add Stone Crafting Table

ModernGamingWorld committed:
Updated API

Updated Forge
Updated JEI
Updated Craft Tweaker
Looks like TOP has an update not sure how to update it

ModernGamingWorld committed:
Merge branch 'NovaMachina-Mods:1.16' into 1.16

remplerus committed:
some fixes for jei and jade

remplerus committed:
fix sound of sieve (again) and add font for see usage of heat sources

until i find out, how to render blocks in jei like in 1.12, that text will show, what kind of block you need to use :)

remplerus committed:
added "no heat source" to top and jade/hwyla + updated api

remplerus committed:
professional copy-pasta from zombieentity to fix #253

NovaMachina committed:
Merge branch '1.16-dev' into 1.16

NovaMachina committed:
Merge pull request #255 from ModernGamingWorld/1.16

Updated APIS & Forge

remplerus committed:
idk why i havent done this earlier xD

NovaMachina committed:
Merge pull request #256 from remplerus/jei-heatsource-rotation

Jei heatsource update

remplerus committed:
make config for stoneblock users

remplerus committed:
add config for enabling makro penalty

remplerus committed:
Update README.md

NovaMachina committed:
Add gradle task to update version file

Since we generate the version number on the fly, it was required that we create a version file on the fly.

[Ticket: AB#60]

NovaMachina committed:
Merge remote-tracking branch 'origin/1.16-dev' into 1.16-dev

NovaMachina committed:
Add script to update version file

[Ticket: AB#60]

NovaMachina committed:
Fix build script

remplerus committed:
remove all

remplerus committed:
add things back

remplerus committed:
fix some small things

NovaMachina committed:
Update build scripts

NovaMachina committed:
Remove debug statement

NovaMachina committed:
Fix blasting recipes

NovaMachina committed:
Fix crook overflow in JEI

[Ticket: AB#32]

remplerus committed:
optimization to api

NovaMachina committed:
Fix # 258 Remove setting player on fire with autoclicker

Removed functionality to set player on fire when an autoclicker was
detected. Prevention is still in place.

[Ticket: 258]

remplerus committed:
Update SieveTile.java

remplerus committed:
Update Config.java

remplerus committed:
Squashed commit of the following:

commit e6f0c3dff5ae44d40348f4a747d1ba7545045dca
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Tue Sep 14 18:30:27 2021 +0200

    fixed bunch of things

    - sieve rendering #262
    - not dropping block when in sieve
    - adding test config for makro issue (NOT FINAL!!!)

commit 8e068ccf1e9aad41182e6e28d4bc6bf3e601559e
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Wed Aug 25 18:59:18 2021 -0500

    Update build scripts

commit 72cb7584a98f04b1ee2939241c89c30f57afcaec
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Tue Aug 24 18:41:28 2021 -0500

    Fix build script

commit cb1e871d36feb7acd225800106a42ff72f0b6ff4
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Tue Aug 24 18:23:53 2021 -0500

    Add script to update version file

    [Ticket: AB#60]

commit 6b5e4a2502ee05ee62a8e403a7953a5e95b47943
Merge: d27da6f cabf7ba
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Tue Aug 24 09:47:42 2021 -0500

    Merge remote-tracking branch 'origin/1.16-dev' into 1.16-dev

commit d27da6f9bad0e719311438f22b599a374ecd7ad0
Author: Jacob Williams <jacob.t.williams@outlook.com>
Date:   Tue Aug 24 09:47:30 2021 -0500

    Add gradle task to update version file

    Since we generate the version number on the fly, it was required that we create a version file on the fly.

    [Ticket: AB#60]

commit cabf7ba479f55625dba3c2a9328ff64c958bd0da
Author: remplerus <44590574+remplerus@users.noreply.github.com>
Date:   Tue Aug 17 23:38:52 2021 +0200

    Update README.md

remplerus committed:
Update SieveTile.java

remplerus committed:
fix #263 + custom crafttweaker crucible fluid showcase

remplerus committed:
add campfires

sam-kirby committed:
Fix #265: Add campfires as a heat source

This pull request adds support for distinguishing if a block is a heat source not only by its resource location, but also by its state. For example, a lit campfire should provide heat to a crucible, but an unlit one should not. This was not previously possible.

This PR also adds lit furnaces and redstone torches to the list of available heat sources

sam-kirby committed:
Refactor: use existant properties local

remplerus committed:
remove campfires for PR #267 as it's better
add grass seed, ancient spore and sweet berries to composting

sam-kirby committed:
Refactor: add an overload heat method to API that takes properties

mjwells2002 committed:
Optimize Crucible isMeltable function by ~10x

remplerus committed:
Update .gitignore

lazynessmind committed:
Include Sweet Berries as a valid compost recipe.

lazynessmind committed:
Improved particles when the barrel is doing the action (compost, fluid transform).

remplerus committed:
updated sweet berries

lazynessmind committed:
Check if the plant is a DoublePlantBlock and then place the upper half. Play sound when placing a seed item.

remplerus committed:
remove stone items: crafting table and stone stick
remove breaking stone / break speed
(moving to own mod)

NovaMachina committed:
Merge pull request #268 from mjwells2002/1.16

Optimize Crucible isMeltable function by ~10x

NovaMachina committed:
Merge pull request #270 from lazynessmind/compost-sweet-berries-recipe

Include Sweet Berries as a valid compost recipe.

NovaMachina committed:
Merge pull request #273 from lazynessmind/fern-bug

DoublePlantBlock check when placing tall plants.

sam-kirby committed:
refactor: make packet structure clearer

remplerus committed:
Merge branch 'small-stoneblock-addon-fix' into remove-stone-utils

remplerus committed:
Merge pull request #11 from remplerus/remove-stone-utils

Remove stone utils

remplerus committed:
Update Config.java

remplerus committed:
remove stone items: crafting table and stone stick
remove breaking stone / break speed
(moving to own mod)

remplerus committed:
Merge remote-tracking branch 'origin/small-stoneblock-addon-fix' into small-stoneblock-addon-fix

NovaMachina committed:
Merge pull request #267 from sam-kirby/1.16

Fix #265: Add campfires as a heat source

remplerus committed:
idk what happened there...

remplerus committed:
update all PR's added for 1.16

remplerus commented:
still need this one for making it final: [#7992](https://github.com/MinecraftForge/MinecraftForge/pull/7992)

NovaMachina committed:
Merge pull request #259 from remplerus/small-stoneblock-addon-fix

[1.16] fix #262 #263 #264 and 2 more things (read comments)

remplerus committed:
update api for addons (no breaking changes!)

NovaMachina committed:
Fix api key issues

NovaMachina committed:
Remove debug

NovaMachina committed:
Merge pull request #277 from remplerus/better_api

update api for addons (no breaking changes!)

remplerus committed:
Merge branch 'NovaMachina-Mods:1.16' into 1.16

remplerus committed:
fix changes made, hopefully don't crash again :|

remplerus committed:
regenerate resources

NovaMachina committed:
Merge pull request #278 from remplerus/1.16

redo changes made in last PR

NovaMachina committed:
Update secrets

remplerus committed:
add enchanting, remove hwyla + their repo

remplerus committed:
optimize imports

NovaMachina committed:
Remove unnecessary files.

NovaMachina committed:
Fix compost overflow in JEI

lazynessmind committed:
Requeted: Revert liquid transformation to the original particles.

metalshark committed:
Added caching, prevented created ItemStacks where possible and simplified use of Streams.

NovaMachina committed:
Merge pull request #271 from lazynessmind/barrel-particles

Less intrusive particles on Barrels.

metalshark committed:
Fixed coding nits for imports.

metalshark committed:
When caching negative outcomes of non-existant recipes or recipe outcomes in registries, clear cache when setting recipes to allow for dynamic loading.

remplerus committed:
- make infested leaves only been broken fast by crooks
- make infested leaves been dropped by shears (again)

remplerus committed:
- make added nether block sounds configable

NovaMachina committed:
Merge pull request #282 from metalshark/1.16

Added caching, prevented created ItemStacks where possible and simplified use of Streams.

NovaMachina committed:
Merge pull request #279 from remplerus/enchantments

add Enchantments to mesh

NovaMachina committed:
Merge pull request #284 from remplerus/infested-leaves-crook

Infested Leaves with drops only breakable by crooks and shears

NovaMachina committed:
Merge pull request #285 from remplerus/make-nether-block-sound-configable

Make added nether block sounds configable

remplerus committed:
- added gravel and dirt to datagen
- optimized sieve datagen
- reloaded resources
- make run-server using other folder than client

Regnander committed:
Create sv_se.json

NovaMachina committed:
Fix #289 Add call to super method in InfestedLeavesBlock

The InfestedLeavesBlock#playerDestroy method was overriding the loot modifier.

[Ticket: AB#106]

NovaMachina committed:
Merge pull request #287 from remplerus/re-add-gravel-coarse-dirt

[1.16] Add back Coarse Dirt to Gravel datagen

NovaMachina committed:
Merge pull request #290 from Regnander/patch-1

Create sv_se.json

remplerus committed:
Merge branch '1.16-dev' of https://github.com/NovaMachina-Mods/ExNihiloSequentia into move-updates-to-1.17-branch

 Conflicts:
	build.gradle
	src/main/java/novamachina/exnihilosequentia/ExNihiloSequentia.java
	src/main/java/novamachina/exnihilosequentia/api/ExNihiloTags.java
	src/main/java/novamachina/exnihilosequentia/api/datagen/AbstractBlockStateGenerator.java
	src/main/java/novamachina/exnihilosequentia/api/datagen/AbstractOreItemGenerator.java
	src/main/java/novamachina/exnihilosequentia/api/utility/Config.java
	src/main/java/novamachina/exnihilosequentia/common/block/BlockSieve.java
	src/main/java/novamachina/exnihilosequentia/common/compat/jei/heat/HeatRecipeCategory.java
	src/main/java/novamachina/exnihilosequentia/common/datagen/ExNihiloBlockStateGenerator.java
	src/main/java/novamachina/exnihilosequentia/common/datagen/ExNihiloDataGenerators.java
	src/main/java/novamachina/exnihilosequentia/common/datagen/ExNihiloFluidTagsGenerator.java
	src/main/java/novamachina/exnihilosequentia/common/datagen/ExNihiloItemTagsGenerator.java
	src/main/java/novamachina/exnihilosequentia/common/datagen/ExNihiloLootTableGenerator.java
	src/main/java/novamachina/exnihilosequentia/common/datagen/ExNihiloRecipeGenerator.java
	src/main/java/novamachina/exnihilosequentia/common/init/ExNihiloBlocks.java
	src/main/java/novamachina/exnihilosequentia/common/init/ExNihiloItems.java
	src/main/java/novamachina/exnihilosequentia/common/item/mesh/EnumMesh.java
	src/main/java/novamachina/exnihilosequentia/common/item/resources/EnumResource.java
	src/main/resources/META-INF/mods.toml
	src/main/resources/assets/exnihilosequentia/lang/en_us.json
	src/main/resources/assets/exnihilosequentia/textures/items/stone_stick.png
	src/main/resources/data/exnihilosequentia/advancements/recipes/exnihilosequentia/ens_crook_stone.json
	src/main/resources/data/exnihilosequentia/recipes/ens_crook_stone.json
	src/main/resources/pack.mcmeta
	update-version-file

remplerus committed:
- Update to 1.18, just waiting for JEI, TOP and resolving fluids

remplerus committed:
- update fluids and packets

remplerus committed:
- update tileentities to use saveAdditional instead of save
- added to 2 classes EXN to avoid collision with forge's renaming
- added 2 TODO's in PacketHandler

remplerus committed:
more updates

- update jade & forge
- made TODO for JEI (as jei isn't available yet)
- fixed wooden barrels to not do their stuff
- undo saveAdditional stuff as it doesn't work rn

NovaMachina commented on README.md:
```diff
@@ -2,7 +2,7 @@
     <a href="https://www.curseforge.com/minecraft/mc-mods/ex-nihilo-sequentia"><img src="http://cf.way2muchnoise.eu/full_400012_downloads.svg" /></a>
     <a href="https://www.curseforge.com/minecraft/mc-mods/ex-nihilo-sequentia"><img src="http://cf.way2muchnoise.eu/versions/400012.svg" /></a>
-    <a href="https://github.com/NovaMachina/ExNihiloSequentia/blob/master/LICENSE"><img alt="GitHub license" src="https://img.shields.io/github/license/NovaMachina/ExNihiloSequentia"></a>
+    <a href="https://github.com/NovaMachina/ExNihiloSequentia/blob/1.16/LICENSE"><img alt="GitHub license" src="https://img.shields.io/github/license/NovaMachina/ExNihiloSequentia"></a>
```
This needs to be updated when we are close to release.

NovaMachina commented on src/main/java/novamachina/exnihilosequentia/api/ExNihiloRegistries.java:
```diff
@@ -1,23 +1,7 @@
-import novamachina.exnihilosequentia.common.registries.HeatRegistry;
-import novamachina.exnihilosequentia.common.registries.SieveRegistry;
+import novamachina.exnihilosequentia.api.registry.*;
+import novamachina.exnihilosequentia.common.registries.*;
```
No bulk imports please.

NovaMachina commented on src/main/java/novamachina/exnihilosequentia/api/utility/Config.java:
```diff
@@ -1,12 +1,13 @@
-package novamachina.exnihilosequentia.common.utility;
+package novamachina.exnihilosequentia.api.utility;
```
This is a concrete class and does not belong in an api package.

NovaMachina commented on src/main/java/novamachina/exnihilosequentia/api/utility/ExNihiloConstants.java:
```diff
@@ -1,4 +1,4 @@
-package novamachina.exnihilosequentia.common.utility;
+package novamachina.exnihilosequentia.api.utility;
```
This is a concrete class and does not belong in an api package.

NovaMachina commented on src/main/java/novamachina/exnihilosequentia/api/utility/ExNihiloLogger.java:
```diff
@@ -1,4 +1,4 @@
-package novamachina.exnihilosequentia.common.utility;
+package novamachina.exnihilosequentia.api.utility;
```
This is a concrete class and does not belong in an API package

NovaMachina commented on src/main/java/novamachina/exnihilosequentia/common/block/BlockSieve.java:
```diff
@@ -115,7 +116,12 @@ public BlockState getStateForPlacement(BlockPlaceContext context) {
         super.playerDestroy(worldIn, player, pos, state, te, stack);
         if (!worldIn.isClientSide() && te instanceof SieveTile) {
-            ((SieveTile) te).removeMesh(false);
+            SieveTile sieveTile = (SieveTile) te;
+            sieveTile.removeMesh(false);
+            if (!sieveTile.getBlockStack().isEmpty()) {
+                worldIn.addFreshEntity(new ItemEntity(worldIn, pos.getX() + 0.5F, pos.getY() + 1.1F, pos
+                        .getZ() + 0.5F, sieveTile.getBlockStack()));
+            }
```
SieveTile#removeMesh already adds the mesh to the world.

remplerus replied:
this doesn't drop mesh, it's dropping the block inside when destroying sieve :)

NovaMachina commented on src/main/java/novamachina/exnihilosequentia/common/block/barrels/AbstractBarrelBlock.java:
```diff
@@ -33,6 +28,8 @@ public BlockEntity newBlockEntity(@Nonnull BlockPos pos, @Nonnull BlockState sta
         return (level, blockPos, blockState, t) -> {
             if (t instanceof StoneBarrelTile tile) {
                 tile.tick();
+            } else if (t instanceof WoodBarrelTile tile) {
+                tile.tick();
```
This could be simplified to checking for an instanceof AbstractBarrelTile

NovaMachina commented on src/main/java/novamachina/exnihilosequentia/common/init/ExNihiloSerializers.java:
```diff
@@ -13,17 +13,9 @@
-import novamachina.exnihilosequentia.common.utility.ExNihiloConstants;
-import novamachina.exnihilosequentia.common.utility.ExNihiloLogger;
+import novamachina.exnihilosequentia.api.utility.ExNihiloConstants;
+import novamachina.exnihilosequentia.common.crafting.serializer.*;
```
No bulk imports please

NovaMachina commented on src/main/java/novamachina/exnihilosequentia/common/item/ore/OreColor.java:
```diff
@@ -1,10 +1,10 @@
 import net.minecraft.world.item.ItemStack;
 
 import javax.annotation.Nonnull;
+import java.awt.*;
```
No bulk imports please.

NovaMachina commented on src/main/java/novamachina/exnihilosequentia/common/registries/SieveRegistry.java:
```diff
@@ -1,27 +1,22 @@
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```
No bulk imports please.

NovaMachina commented on src/main/java/novamachina/exnihilosequentia/common/tileentity/barrel/mode/BarrelModeRegistry.java:
```diff
@@ -1,14 +1,10 @@
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```
No bulk imports please

NovaMachina reviewed (changes requested):
I noticed that a lot of asset files got removed, is this intentional?

Also, some of the things that you were changing got me thinking about restructuring some things in the mod. I'll see what I can do, but looks good otherwise!

remplerus committed:
- update bulk imports (again :| )
- made changes requested

remplerus committed:
- update gradle to 7.3 and forge to 38.0.10
- readd PacketHandler due to fix of forge
- retry nbt's on tiles :|

remplerus committed:
Merge branch '1.16' into move-updates-to-1.18-branch

# Conflicts:
#	build.gradle
#	src/main/java/novamachina/exnihilosequentia/api/registry/ICrucibleRegistry.java
#	src/main/java/novamachina/exnihilosequentia/common/block/InfestedLeavesBlock.java
#	src/main/java/novamachina/exnihilosequentia/common/block/barrels/NetherBarrelBlock.java
#	src/main/java/novamachina/exnihilosequentia/common/block/crucibles/NetherCrucibleBlock.java
#	src/main/java/novamachina/exnihilosequentia/common/block/sieves/NetherSieveBlock.java
#	src/main/java/novamachina/exnihilosequentia/common/compat/jei/crucible/FiredCrucibleTooltipCallback.java
#	src/main/java/novamachina/exnihilosequentia/common/compat/jei/crucible/WoodCrucibleTooltipCallback.java
#	src/main/java/novamachina/exnihilosequentia/common/datagen/ExNihiloRecipeGenerator.java
#	src/main/java/novamachina/exnihilosequentia/common/item/mesh/MeshItem.java
#	src/main/java/novamachina/exnihilosequentia/common/item/tools/crook/CrookBaseItem.java
#	src/main/java/novamachina/exnihilosequentia/common/registries/CompostRegistry.java
#	src/main/java/novamachina/exnihilosequentia/common/registries/CrookRegistry.java
#	src/main/java/novamachina/exnihilosequentia/common/registries/CrucibleRegistry.java
#	src/main/java/novamachina/exnihilosequentia/common/registries/FluidItemTransformRegistry.java
#	src/main/java/novamachina/exnihilosequentia/common/registries/FluidOnTopRegistry.java
#	src/main/java/novamachina/exnihilosequentia/common/registries/FluidTransformRegistry.java
#	src/main/java/novamachina/exnihilosequentia/common/registries/HeatRegistry.java
#	src/main/java/novamachina/exnihilosequentia/common/registries/SieveRegistry.java
#	src/main/java/novamachina/exnihilosequentia/common/tileentity/SieveTile.java
#	src/main/java/novamachina/exnihilosequentia/common/tileentity/barrel/mode/CompostBarrelMode.java
#	src/main/java/novamachina/exnihilosequentia/common/tileentity/barrel/mode/FluidTransformBarrelMode.java
#	src/main/java/novamachina/exnihilosequentia/common/utility/Config.java
#	src/main/resources/data/exnihilosequentia/recipes/sieve/ens_dirt.json

remplerus committed:
- add optimization and additions from 1.16 to 1.18

remplerus committed:
- enabled JEI and TOP compat

NovaMachina commented on build.gradle:
```diff
@@ -88,17 +88,8 @@ processResources {
     finalizedBy replaceResources
 }
 
-task updateVersion() {
-    doLast {
-        new File(projectDir, "update.json").text =
-"""{
-    "homepage": "https://www.curseforge.com/minecraft/mc-mods/ex-nihilo-sequentia",
-    "promos": {
-        "1.17-latest": "${version}",
-        "1.17-recommended": "${version}"
-    }
-}"""
-    }
```
This should not be removed.

remplerus replied:
this isn't removed, that's one of your optimizations as createVersionFile -> CreateVersionClass doing same as updateVersion

NovaMachina reviewed (changes requested):
One last item in build.gradle

remplerus committed:
- re-added resources (from datagen)

NovaMachina commented:
Looks like there are some merge conflicts. Please resolve these.

remplerus committed:
test

remplerus commented:
now it should be mergable

remplerus committed:
- reenable crafttweaker
- try to do make "blockentities" work (new name of tileentities) doesn't show up their work ig :'(

remplerus committed:
- try to do make "blockentities" work (new name of tileentities) doesn't show up their work ig :'(

remplerus commented:
closing this PR to have as here are too many bugs :(
